daceml/onnx/onnx_importer.py

```diff
@@ -298,6 +301,32 @@ def __init__(self,
                         dace.Memlet.from_array(clean_onnx_name(name),
                                                data_desc))
 
+        # insert copies from outputs to __return arrays
+        copy_out_state = self.sdfg.add_state_after(self.state,
+                                                   label='copy_out')
+        new_output_names = []
+        for i, output in enumerate(self.outputs):
+            clean_name = clean_onnx_name(output)
+            new_output_name = '__return'
+            if len(self.outputs) > 1:
+                new_output_name += '_' + str(i)
+            new_output_names.append(new_output_name)
+
+            # insert new descriptor
+            self.sdfg.arrays[new_output_name] = copy.deepcopy(
+                self.sdfg.arrays[clean_name])
+            self.sdfg.arrays[new_output_name].transient = False
+
+            copy_out_state.add_edge(copy_out_state.add_read(clean_name), None,
+                                    copy_out_state.add_write(new_output_name),
+                                    None,
+                                    self.sdfg.make_array_memlet(clean_name))
+
+        # finally, rename outputs
+        self.outputs = new_output_names
+
+        self.sdfg.apply_transformations_repeated(interstate.StateFusion)
```

Comment:
Why do you need to do this? Also, there's a faster, safer FuseStates pass

Reply:
Switched to `fuse_states`. I can't use the `FuseStates` pass since daceml is still on `0.13.3`